apps/blueman-adapters.in

```diff
@@ -19,22 +20,21 @@ _dirname = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
 if 'BLUEMAN_SOURCE' in os.environ:
     sys.path.insert(0, _dirname)
 
-from blueman.Functions import *
+from blueman.Functions import set_proc_title
 from blueman.main.Adapter import BluemanAdapters
 
 # Workaround introspection bug, gnome bug 622084
 signal.signal(signal.SIGINT, signal.SIG_DFL)
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--socket-id", dest="socket_id", action="store", type=int, metavar="ID")
+    parser.add_argument("adapter", nargs="*", metavar="ADAPTER NAME")
```

Comment:
`nargs="*"`? I think that allows multiple values and returns a list which `BluemanAdapters` does not expect.